src/systems/include/mp-units/systems/si/unit_symbols.h

```diff
@@ -474,6 +474,32 @@ inline constexpr auto YF = yotta<farad>;
 inline constexpr auto RF = ronna<farad>;
 inline constexpr auto QF = quetta<farad>;
 
+inline constexpr auto qohm = quecto<si::ohm>;
+inline constexpr auto rohm = ronto<si::ohm>;
+inline constexpr auto yohm = yocto<si::ohm>;
+inline constexpr auto zohm = zepto<si::ohm>;
+inline constexpr auto aohm = atto<si::ohm>;
+inline constexpr auto fohm = femto<si::ohm>;
+inline constexpr auto pohm = pico<si::ohm>;
+inline constexpr auto nohm = nano<si::ohm>;
+inline constexpr auto uohm = micro<si::ohm>;
+inline constexpr auto mohm = milli<si::ohm>;
+inline constexpr auto cohm = centi<si::ohm>;
+inline constexpr auto dohm = deci<si::ohm>;
+inline constexpr auto ohm = si::ohm;
```

Comment:
I think that we can go with a using-declaration here to not copy the same identifiers to two namespaces:

```suggestion
using si::ohm;
```

Reply:
Ah right, didn't think of that!